pom.xml

```diff
@@ -21,51 +22,42 @@
   </scm>
 
     <properties>
-        <jenkins.version>1.532</jenkins.version> <!-- ParameterDefinition#getFormattedDescription is since 1.520 -->
-        <jenkins-test-harness.version>1.532</jenkins-test-harness.version>
-        <java.level>6</java.level>
+        <jenkins.version>1.625.3</jenkins.version>
```

Comment:
1.651.3 might be better for SECURITY-170 affects much.

Reply:
Anyway I saw no 170-related test failures.

Reply:
I decided to target 1.651.3 as in #24 .
* The next release anyway updates the target version from 1.532.
* This plugin will depend on matrix-project-1.7 for compatibility tests pass SECURITY-170, and it looks natural to me to target core with SECURITY-170.
* It makes easier to see XMLs created by SECURITY-170.
    * Of course, I know it will be fixed in jenkinsci/jenkins#3312 .